packages/dashboard/src/components/Card/Chart/Chart.tsx

```diff
@@ -142,6 +140,8 @@ export class Chart extends React.Component<IChartInternalProps, { _width: number
   private _getLineChart = (): JSX.Element => {
     const { chartData, timeRange } = this.props;
     let dateDataType = false;
+    const getClassNames = classNamesFunction<IChartProps, IChartStyles>();
```

Comment:
This should preferably be at the top of the file rather than redefined in every call to _getLineChart.

Reply:
updated to top of file